docs/sql-programming-guide.md

```diff
@@ -1808,8 +1807,7 @@ df = sqlContext.load(source="jdbc", url="jdbc:postgresql:dbserver", dbtable="sch
 
 {% highlight r %}
 
-df <- loadDF(sqlContext, source="jdbc", url="jdbc:postgresql:dbserver", dbtable="schema.tablename")
-
+df <- read.df(sqlContext, source = "jdbc",url="jdbc:postgresql:dbserver", dbtable="schema.tablename")
```

Comment:
you are correct I could not find any reference that loadDF is deprecated. I will revert this change back.